watertap/costing/watertap_costing_package.py

```diff
@@ -96,6 +96,14 @@ def build_process_costs(self):
                 + sum(self.aggregate_flow_costs.values()) * self.utilization_factor
             )
 
+        self.total_annualized_cost = pyo.Expression(
+            expr=(
+                self.total_capital_cost * self.capital_recovery_factor
```

Comment:
Similar to my earlier comments for future work, with regard to a breakdown of LCOW cost components, would be great to have an `annualized_cost` attribute for every capital costs. With that, all we'd need to do is then divide by the desired flowrate.